src/composable/chat/useChatModeration.ts

```diff
@@ -87,7 +85,18 @@ export function useChatModeration(ctx: ChannelContext, victim: string) {
 	}
 
 	function deleteChatMessage(msgID: string) {
-		messages.sendMessage(`/delete ${msgID}`);
+		const apollo = useApollo();
```

Comment:
Looks good, just one thing the `useApollo` call should probably be moved outside of this function and into the parent scope, the same way the `useChatMessages` composable was previously preserved.